sections/earn/Incentives.tsx

```diff
@@ -102,6 +103,7 @@ const Incentives: FC<IncentivesProps> = ({
 							now,
 							route: ROUTES.Earn.Curve_LP,
 							tab: Tab.Curve_LP,
+							needsToSettle: false,
```

Comment:
same comment as above for the props you pass in here. doesn't need this field for all entries